core/src/main/java/org/eclipse/hono/util/TenantConstants.java

```diff
@@ -60,9 +60,9 @@
      */
     public static final String FIELD_PAYLOAD_PUBLIC_KEY = "public-key";
     /**
-     * The name of the property that contains the trusted certificate authority configured for a tenant.
+     * The name of the property that contains the list of trusted certificate authorities configured for this tenant.
```

Comment:
> ... for a tenant.

In the constants, there is no *this tenant*, is there?

Reply:
i like this comment :-)